index.bs

```diff
@@ -359,36 +415,518 @@ To <dfn>sanitizeToString</dfn> a given |input|, run these steps:
 
 ## Default Configuration {#defaults}
 
-Issue: The sanitizer defaults need to be carefully vetted, and are still
-    under discussion. The values below are for illustrative purposes only.
+Issue(WICG/sanitizer-api#38): The sanitizer defaults need to be carefully
+    vetted, and are still under discussion. The values below are for
+    illustrative purposes only.
 
 The sanitizer has a built-in default configuration, which aims to eliminate
 any script-injection possibility. Note that the [=sanitize document fragment=]
 algorithm
 is defined so that these defaults are handled first and cannot be overriden
 by a custom configuration.
 
+Note: The default lists are expected to change when user agents support new
+    elements or attributes, with regard to only those new elements or attributes,
+    without an explicit re-release of this specification.
+
+: Builtin Drop Elements
+
+:: The <dfn>built-in element allow list</dfn> has the following value:
+    ```
+    [
+          "a", "abbr", "acronym", "address", "area", "article", "aside", "audio",
```

Comment:
Nit: For the sake of "git blame" and maintenance, is there an easy way to list them line-by-line? Maybe we can use [bikeshed's include file feature](https://tabatkins.github.io/bikeshed/#including)?

Reply:
Done. This is indeed much nicer.